clients/scheduler/src/index.html

```diff
@@ -77,8 +77,9 @@
           Scheduler.init({
             api: apiBaseUrl,
             appointmentCoding: {
-              code: 'CODE',
+              code: '308335008',
```

Comment:
The fact that the scheduler worked with the old configuration (`code: 'CODE'`) before and now it does not, leads me to believe that Canvas updated their API to run some validation against the `Code` that is passed. 

Not 100% though.

Reply:
This seems reasonable, assuming that this line is the primary fix? Is it worth trying to send through the actual appointment code from Jade?

Reply:
This is just the configuration used to test the scheduler locally. We are sending the appropriate reason code from Jade FWIW.

Reply:
Right, didn't take note of file name before